packages/devtools_app/lib/src/profiler/cpu_profiler.dart

```diff
@@ -225,7 +229,12 @@ class _CpuProfilerState extends State<CpuProfiler>
       },
     );
     // TODO(kenz): make this order configurable.
-    return [bottomUp, callTree, cpuFlameChart];
+    return [
+      if (widget.summaryView != null) widget.summaryView,
+      widget.data.isEmpty ? _buildEmptyDataView() : bottomUp,
```

Comment:
it would be nice to not show these tabs at all rather than showing them with empty data when they are not available.

Reply:
done